LICENSE

```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
+
+Copyright (c) 2022 Löffelhardt GmbH & Co.KG.
```

Comment:
Markus Thielen, Löffelhardt GmbH & Co.KG. and contributors

Reply:
Yes, I forgot it.